pkg/kv/kvserver/concurrency/concurrency_manager.go

```diff
@@ -131,7 +131,7 @@ var UnreplicatedLockReliabilityLeaseTransfer = settings.RegisterBoolSetting(
 	settings.SystemOnly,
 	"kv.lock_table.unreplicated_lock_reliability.lease_transfer.enabled",
 	"whether the replica should attempt to keep unreplicated locks during lease transfers",
-	metamorphic.ConstantWithTestBool("kv.lock_table.unreplicated_lock_reliability.lease_transfer.enabled", true),
+	metamorphic.ConstantWithTestBool("kv.lock_table.unreplicated_lock_reliability.lease_transfer.enabled", false),
```

Comment:
Should we remove the metamorphic constant as well, at least for now? That way, we won't have to triage KVNemesis failures on this branch that could be similar to https://github.com/cockroachdb/cockroach/issues/145164?

Reply:
Ah good idea. Will force push shortly.